controllers/batterycontroller.py

```diff
@@ -1,16 +1,6 @@
-from TestData import*
-from canBus import*
-from Controller import*
-class BatteryController(Controller):
+class BatteryController():
+  testData
```

Comment:
See the first comment on backupcontroller.py with respect to lines 2 and 3 here.